test/utils/TestUtils.sol

```diff
@@ -0,0 +1,205 @@
+    // SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
```suggestion
// SPDX-License-Identifier: AGPL-3.0-or-later
```

Reply:
Implemented in https://github.com/makerdao/dss-allocator/pull/15